app/experimenter/experiments/constants/nimbus.py

```diff
@@ -445,7 +551,23 @@ class ConclusionRecommendation(models.TextChoices):
         STOP = "STOP", "Stop"
         FOLLOWUP = "FOLLOWUP", "Run follow ups"
 
-    Application = Application
+    ROLLOUT_SUPPORT = (
+        RolloutSupport(
+            application=Application.DESKTOP,
+            channel=Channel.RELEASE,
+            firefox_min_version=Version.FIREFOX_97,
+        ),
+        RolloutSupport(
+            application=Application.DESKTOP,
+            channel=Channel.NIGHTLY,
+            firefox_min_version=Version.FIREFOX_97,
+        ),
+        RolloutSupport(
+            application=Application.FENIX,
+            channel=Channel.RELEASE,
+            firefox_min_version=Version.FIREFOX_95,
+        ),
+    )
```

Comment:
Why do we have to break this down by channel?  Version should be sufficient shouldn't it?

Reply:
Yeah, I wasn't sure about that. But also yeah, I guess versions are what ride the trains between the channels, so that does sound redundant.

I also wasn't sure about these exact versions - part of why I kept it as a warning rather than an error blocking review for launch. Could hold these changes back, or just merge with a best guess and update as soon as we know for sure about versions.